src/AvaloniaEdit/Editing/LineNumberMargin.cs

```diff
@@ -68,6 +68,8 @@ protected override Size MeasureOverride(Size availableSize)
         /// <inheritdoc/>
         public override void Render(DrawingContext drawingContext)
         {
+            EmSize = GetValue(TextBlock.FontSizeProperty);
```

Comment:
Move inside `if` block below

Reply:
I believe I found a better solution. Now it updates the fontsize and the margin correctly